.github/stale.yml

```diff
@@ -15,4 +15,7 @@ issues:
         - bug
     markComment: >
         This issue has been automatically marked as stale because it has not had
-        recent activity. It will be closed if no further activity occurs.
+        any activity in the last 2 years. If you feel that this is a valid issue,
+        just comment and the stale tag would be removed, otherwise it will be
```

Comment:
will be removed?

Reply:
I was actually looking for the stale bot configuration to see how this works. @rschlussel - do you know what the correct wording should be?

Reply:
also, semicolon instead of a comma.

Reply:
@mayankgarg1990 I don't understand your question, but I like Rongrong's wording.  For your reference here's the stalebot repo: https://github.com/probot/stale.